src/libYARP_sig/src/Image.cpp

```diff
@@ -298,44 +297,42 @@ void ImageStorage::_set_ipl_header(int x, int y, int pixel_type, int quantum,
     int origin = topIsLow?IPL_ORIGIN_TL:IPL_ORIGIN_BL;
     int implemented_yet = 1;
     // used to allocate the ipl header.
+#define iplCreateImageHeaderMono   pImage = iplCreateImageHeader( 1, 0, IPL_DEPTH_8U,  (char *)"GRAY", (char *)"GRAY", IPL_DATA_ORDER_PIXEL, origin, quantum, x, y, nullptr, nullptr, nullptr, nullptr)
+#define iplCreateImageHeaderMono16 pImage = iplCreateImageHeader( 1, 0, IPL_DEPTH_16U, (char *)"GRAY", (char *)"GRAY", IPL_DATA_ORDER_PIXEL, origin, quantum, x, y, nullptr, nullptr, nullptr, nullptr)
```

Comment:
Please remove this macro, I don't want to see new macro magic added to a code that is already almost unreadable...

Reply:
no macro magic here.. it is simply the method already called many times in that portion of code.. in my case i've added 8 types that requires the method called with the same parameters (4 and 4).. are you sure the code to be more readable removing these macro?.. cause in this case i think it is time for refactoring..

Reply:
I do agree with @drdanz, defines shall not have to be used to simply _rename_ things. That's a bad practise and does not help readability, please stop doing it. Let the code speak for itself

Reply:
in this case are used for not repeating code and connecting the same code under a common flag, to let the reader know that all the involved cases calls the same function.. and this concept is not as clear when you call a method with 14 parameters! but since you insist, lets refactor!